lib/gtl_formatter.rb

```diff
@@ -177,9 +177,19 @@ def self.os_info_details(item)
     "svg/os-#{item.os_image_name}.svg"
   end
 
+  def self.item_provider(item)
+    type = item.class.to_s.split('::').last
+    case type
+    when 'ResourcePool', 'Container'
```

Comment:
That being said, I don't know why we are calling out these 2 classes specifically?  I would expect the right way to fix this would be to implement the ext_management_system method directly on the item or however it works the the majority of other items.

Reply:
`item.decorate` is returning this -
![image](https://github.com/ManageIQ/manageiq-ui-classic/assets/87487049/8f69535f-ec1b-486e-b1d6-c72eccb8feb1)


and `item.ext_management_system.decorate` will give this. (notice the icon changes)
![image](https://github.com/ManageIQ/manageiq-ui-classic/assets/87487049/ab2c1e8b-60c6-449e-890e-18b40dcd3ffe)